flink-end-to-end-tests/flink-streaming-kinesis-test/pom.xml

```diff
@@ -65,6 +71,13 @@ under the License.
 			<version>${junit.version}</version>
 			<scope>compile</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+			<version>1.16.22</version>
+			<scope>provided</scope>
+		</dependency>
```

Comment:
As the Lombok dependency is not widely used in Flink, does it bring enough value to pull it in for a single test? Have we consulted Lombok usage with Flink community ?

Reply:
Yeah good shout, I will remove.